examples/dendrogram.py

```diff
@@ -135,6 +155,31 @@ def dendrogram(tree_object, show_diameters=False, new_fig=True, subplot=False, *
 
     linewidths = linewidths / min(linewidths) if show_diameters else 1
 
+    xlabel = 'Length (um)'
+    ylabel = ''
+
+    if rotation == 'left':
+
+        angle = pi
+
+        affine2D_transfrom(positions, cos(angle), -sin(angle), sin(angle), cos(angle))
```

Comment:
This line is repeated in 3 places. I think it can be moved out of the if-else blocks.